user-guide/Advanced_Modules/DOM/Tutorials/DOM_Remove_Enum_Entry.md

```diff
@@ -0,0 +1,406 @@
+            }
+
+            // Create of the actual DomInstance
+            var domInstance = new DomInstance()
+            {
+                DomDefinitionId = domDefinitionId,
+                Sections = { generalSection }
+            };
+            domInstance = _domHelper.DomInstances.Create(domInstance);
+        }
+    }
+}
+```
+
+To visualize the setup, create a low code app. On that low code app:
+1. Drag the DOM definition: *"Object Manager Definitions" > "people_app" > "PersonDomDefinition"*
+1. Choose the "Form" visualization
+1. Create a query that shows all instances: *"Queries" > "Get object manager instances" > "people_app" > "PersonDomDefinition"*
+1. Drag the query and choose "Table" visualization
+
+   ![Step1](~/user-guide/images/DOM_Remove_Enum_Entry_Step1_1.png)
+
+   ![Step1_2](~/user-guide/images/DOM_Remove_Enum_Entry_Step1_2.png)
+
+## Step 2: Soft-delete the enum entry
+
+The given code does the following:
+1. Gets the enum entry from the DOM definition.
+1. Sets the property "IsSoftDeleted" of the enum entry to true.
+1. Updates the DOM definition.
+
+```C#
+using System;
+using System.Linq;
+using Skyline.DataMiner.Automation;
+using Skyline.DataMiner.Net.Apps.DataMinerObjectModel;
+using Skyline.DataMiner.Net.ManagerStore;
+using Skyline.DataMiner.Net.Sections;
+
+
+namespace Tutorial
+{
+    ///<summary>
+    /// This script will set the "IsSoftDeleted" property of the enum entry "Director" of the DOM definition.
+    ///</summary>
+    public class Script
+    {
+        private const string ModuleName = "people_app";
+        private DomHelper _domHelper;
+        private Engine _engine;
+
+        public void Run(Engine engine)
+        {
+            _engine = engine;
+            _domHelper = new DomHelper(_engine.SendSLNetMessages, ModuleName);
+
+            // Searching by name, can also use the IDs, it will do less calls and needs less null checks
+            var sectionDefinitionName = "PeopleSectionDefinition";
+            var fieldDescriptorName = "JobTitle";
+            var enumName = "Director";
+
+            var sectionDefinitions = _domHelper.SectionDefinitions.ReadAll();
+            var sectionDefinition = sectionDefinitions.First(x => x.GetName() == sectionDefinitionName);
+            var fieldDescriptor = sectionDefinition.GetAllFieldDescriptors().First(x => x.Name == fieldDescriptorName) as GenericEnumFieldDescriptor;
+            if (fieldDescriptor == null) { return; }
+            var enumDelete = fieldDescriptor.GenericEnumInstance.Entries.First(x => x.DisplayName == enumName);
+            enumDelete.IsSoftDeleted = true;
+            _domHelper.SectionDefinitions.Update(sectionDefinition);
+        }
+    }
+}
+```
+
+This example uses the names of the section definition and field descriptor. Another option is to use the IDs, that way less calls and null checks are needed, but those IDs still need to be looked up. Note: these IDs will not work, you need the IDs of your setup.
```

Comment:
Maybe this can be left out here if you are using the filter element. We only do one call to the server. Whether this is a name filter or ID filter doesn't really matter (enough).